src/main/java/org/ccci/util/mail/MimeMailMessage.java

```diff
@@ -0,0 +1,206 @@
+package org.ccci.util.mail;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Throwables;
+import com.google.common.collect.Lists;
+import java.util.Arrays;
+import java.util.List;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.SendFailedException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import org.apache.log4j.Logger;
+
+/**
+ * <p>Simple API for emails. Sends either a separate email to each person that has been added via
+ * {@link #addTo(InternetAddress...)}, or a single email to all recipients, depending on whether
+ * {@link #sendToEach()} or {@link #sendToAll()} is called.
+ *
+ * <p>Adapted from old framework jar. Use by injecting a {@link MimeMailMessageFactory}, and calling
+ * {@link MimeMailMessageFactory#createApplicationMessage()}.
```

Comment:
Is this new code in fact still considered adapted from the 'old framework jar'?  You might consider removing that since not directly adapted.